openrr-apps/src/robot_config.rs

```diff
@@ -17,6 +17,9 @@ pub struct RobotConfig {
     pub ros_clients_configs: Vec<RosControlClientConfig>,
     pub urdf_viz_clients_configs: Vec<UrdfVizWebClientConfig>,
 
+    #[cfg(feature = "ros")]
+    pub ros_espeak_client_config: Option<RosEspeakClientConfig>,
+
```

Comment:
I think that it is OK to exist ros_***_config even if ros feature is off. (If it does not cause compile error or some problems)

Reply:
RosEspeakClientConfig is defined in arci-ros.
So I thinks we cloud not compile without feature=ros